src/Algebra/Solver/CommutativeMonoid/Example.agda

```diff
@@ -9,10 +9,8 @@
 module Algebra.Solver.CommutativeMonoid.Example where
 
 open import Relation.Binary.PropositionalEquality.Core using (_≡_)
-
```

Comment:
Given that this is an `Example` module, please don't delete blank lines. 
I think that that is going above-and-beyond 'import cleaning', as I'm guessing they were originally put in for didactic intent/purposes?